src/wallet/wallet2.cpp

```diff
@@ -2289,15 +2287,15 @@ void wallet2::load(const std::string& wallet_, const std::string& password)
 
       std::stringstream iss;
       iss << cache_data;
-      boost::archive::binary_iarchive ar(iss);
+      cereal::PortableBinaryInputArchive ar(iss);
```

Comment:
The binary formats are not compatible, so what happens when a user with an existing wallet opens the file? A conversion tool or supporting multiple input formats may be needed. Output should be fine (transparent upgrade).